README.md

```diff
@@ -22,7 +22,7 @@ A modern web-based interface for building Maestro agents and workflows using AI
      python3 -m venv .venv
      source .venv/bin/activate
      ```
-   - Install required dependencies (this will install everything, including Maestro and DuckDuckGo):
+   - Install required dependencies:
      ```bash
      pip install -r api/requirements.txt
```

Comment:
not relevant for this PR, but at some point we should probably be using uv for this I'd assume...